spec/squid_spec.rb

```diff
@@ -6,7 +6,7 @@
   let(:data) { {} }
   let(:options) { {legend: false, baseline: false, steps: 0, format: :currency} }
   let(:settings) { options }
-  let(:blue_rgb) { [0.1804, 0.3412, 0.549] }
+  let(:blue_rgb) { %w[2e 57 8c].map { |c| (c.hex / 255.0).round(4) } }
```

Comment:
What is this change doing? Can you explain how the behavior changes here? It seems like we could have a potential breaking change.

Reply:
I think this line in particular was an artifact of some debugging I was doing on this expectation:

https://github.com/nullscreen/squid/pull/68/files#diff-48f2303afd24f9a0a1300d5d89de1dbb45d2c81efaefe053e3814a46fcfa172fR43

`colors_of` calls PDF inspector which introduces a little rounding error.

```
  1) Prawn::Document#chart given one series has blue columns
     Failure/Error: expect(colors_of(chart).fill_color).to eq blue_rgb
     
       expected: [0.1804, 0.3412, 0.549]
            got: [0.18039, 0.34118, 0.54902]
     
       (compared using ==)
     # ./spec/squid_spec.rb:44:in `block (3 levels) in <top (required)>'
```

In any case, I've reverted the line you've highlighted and the specs pass.